core/templates/pages/exploration-editor-page/exploration-editor-page.component.ts

```diff
@@ -196,21 +204,23 @@ angular.module('oppia').component('explorationEditorPage', {
         ExplorationStatesService, ExplorationTagsService,
         ExplorationTitleService, ExplorationWarningsService,
         FocusManagerService, GraphDataService, InternetConnectivityService,
-        LoaderService, NgbModal, PageTitleService, ParamChangesObjectFactory,
+        LoaderService, LocalStorageService, NgbModal,
+        PageTitleService, ParamChangesObjectFactory,
         ParamSpecsObjectFactory, PreventPageUnloadEventService,
         RouterService, SiteAnalyticsService,
         StateClassifierMappingService,
         StateEditorRefreshService, StateEditorService,
         StateTutorialFirstTimeService,
         ThreadDataBackendApiService,
         UserEmailPreferencesService, UserExplorationPermissionsService,
-        UserService, WindowDimensionsService) {
+        UserService, WindowDimensionsService, WindowRef) {
       var ctrl = this;
       var reconnectedMessageTimeoutMilliseconds = 4000;
       var disconnectedMessageTimeoutMilliseconds = 5000;
       ctrl.directiveSubscriptions = new Subscription();
       ctrl.autosaveIsInProgress = false;
       ctrl.connectedToInternet = true;
+      ctrl.isExplorationInitialized = false;
```

Comment:
```suggestion
      ctrl.explorationIsInitialized = false;
```

Reply:
Also, if this is not used in the HTML, it need not be attached to the ctrl scope. Ditto below for the functions.

Reply:
I will be making the changes on other PRs ASAP.